### ISSUE TITLE
Type on Typescript

### ISSUE THREAD

**Issue by rahman77889** [Midtrans/midtrans-nodejs-client#59]
Hi Midtrans team

I have built some types to use this library in Typescript, 
because I couldn't use this library on typescript, now I could do it

this is my repo on git
[github](https://github.com/rahman77889/type-midtrans-client)

and repo on npm
[npm](https://www.npmjs.com/package/@abdulrahmanreza/midtrans-client)

Please mention me if need

**Comment by Xaxxis** [comment]
Hi @rahman77889, We are grateful for your willingness to share your knowledge and expertise. To maintain the quality and integrity of our open-source community, please explain in more detail about your project and how to use it. Your project only has one type file for type response, constructor, etc. Without any sample of how to use in typescript and unit tests for your code. 

Please add more details on your project about how to use it, and add the unit test. So our community can better understand your project.

These are contributed by open-source contributors not maintained by Midtrans, which may provide good alternatives if the official library list doesn't meet your needs.
https://docs.midtrans.com/docs/midtrans-api-libraries-plugins#community-contributed

Thank You

**Comment by Xaxxis** [comment]
Closing this issue due to no response from the author. If any general questions, don't hesitate to get in touch with us [here](https://midtrans.com/id/contact-us) or feel free to reopen this issue if the author becomes available to provide further details.

Thank You

**Comment by rahman77889** [comment]
> Hi @rahman77889, We are grateful for your willingness to share your knowledge and expertise. To maintain the quality and integrity of our open-source community, please explain in more detail about your project and how to use it. Your project only has one type file for type response, constructor, etc. Without any sample of how to use in typescript and unit tests for your code.
> 
> Please add more details on your project about how to use it, and add the unit test. So our community can better understand your project.
> 
> These are contributed by open-source contributors not maintained by Midtrans, which may provide good alternatives if the official library list doesn't meet your needs. https://docs.midtrans.com/docs/midtrans-api-libraries-plugins#community-contributed
> 
> Thank You

Hai sorry for slow response, 

this kind of project is not such as midtrans SDK package, this project is to define if you using typescript it will require @types package to use the npm package

but until now there's no type of package for package SDK official on https://www.npmjs.com/package/midtrans-client

then I decided to create a new one to solve this problem

You still need the official package SDK on https://www.npmjs.com/package/midtrans-client

So basically this project's help your official SDK to use in typescript language
and also don't need any sample of how to use it in typescript and unit tests because we still use your official SDK, again this project only helps the official SDK to use in typescript using @types to get general response from every API from official SDK

refer to the detailed installation process it's already in the description of NPM description or GitHub description at https://www.npmjs.com/package/@abdulrahmanreza/midtrans-client

Thank You
